gradle.properties

```diff
@@ -1 +1,3 @@
-org.gradle.jvmargs=-Xmx1536M
+android.nonFinalResIds=false
+android.nonTransitiveRClass=false
+org.gradle.jvmargs=-xmx4608m
```

Comment:
Both build options were required due to the update to Android Gradle Plugin 8.0. Both these options were the old default values (https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#default-changes). Removing them results in a compilation error. So I fixed the issue in `PopupActicity.java` where `R.id.fullscreen` was no longer constant. So replacing the switch-statement which had only a single case with an if-statement fixed the compilation error.

Reply:
My issue is only about the last line: the first X has to be upper case:

```
Unrecognized option: -xmx4608m
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

Reply:
Oh alright. Now I see the difference in your suggested change. Interessting that AndroidStudio suggested to add exactly this line as it was commited. I don't have any problems with it. But you are absolutly right, the option starts with an upper case X. I changed it.
I leave the other changes in, because they were only minor ones.